server/policies/collection.js

```diff
@@ -14,6 +14,8 @@ allow(User, ['read', 'export'], Collection, (user, collection) => {
   if (
     collection.private &&
     (!collection.memberships || !collection.memberships.length)
+    // XXX: does this need the invariant that is below in publish/update?
+    // it doesn't seem to check the permission type
```

Comment:
@tommoor looks like this is just checking for the presence of a preloaded membership at all, which I think is ok, but should this have the same invariant as https://github.com/outline/outline/pull/1102/files#diff-240dcfa58e2e777bcc9ac5081c588bfaR30

Reply:
I believe there's a reason why, but I don't remember – possibly something to do with allowing read permissions for sharing.